fixtures.ttar

```diff
@@ -4879,7 +4879,7 @@ Mode: 664
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 Path: fixtures/sys/class/nvme/nvme0/model
 Lines: 1
-Samsung SSD 970 PRO 512GB               
+Samsung SSD 970 PRO 512GB
```

Comment:
It looks like you're re-writing unrelated fixtures. Some of these fixtures have intentional extra whitespace and other things since the source kernel files have them.

Please only change fixtures that are related to your change.